unit_tests/test_cassandra_stress_thread.py

```diff
@@ -141,3 +141,33 @@ def cleanup_thread():
 
     assert "latency 99th percentile" in output[0]
     assert float(output[0]["latency 99th percentile"]) > 0
+
+
+@pytest.mark.parametrize("compressor, cql_compression", [
+    ("Deflate", "none"), ("LZ4", "lz4"), ("Snappy", "snappy"), ("Zstd", "none")
```

Comment:
I didn't know we supported deflate for CQL compression? I think we are mixing sstable compression and CQL protocol level compression?

Reply:
Here CQL compression for Deflate and ZStd is none, so there will be no compression in those cases, and the first compression is for Scylla (schema level) and the later one is compression for the protocol, which tipped me off guard when first checking.

There is a `'` after Compressor making it a part of -schema argument

Reply:
yes it's mixing it, the issue we had was with cql compression, that only support snappy / lz4